docs/reference/api/security/TLS.md

```diff
@@ -41,6 +41,26 @@ Then create a file named `mbed_app.json` at the root of your application with th
 }
 ```
 
+### Mbed TLS platform context
+
+Some hardware accelerators require initialization, regardless of the specific cryptography engine. For this, we introduced `mbedtls_platform_setup()` and `mbedtls_platform_terminate()`.
+
+As the [examples](#mbed-tls-examples) show, you *must* call the `mbedtls_platform_setup()` function before you call any Mbed TLS API. After using the Mbed TLS API, you *must* call `mbedtls_platform_teardown()` to terminate the platform hardware driver. The platform context parameter for these function *should* be `NULL`.
```

Comment:
Query: When we say "should", do we mean if everything's working correctly, the parameter is NULL? Or do we mean something else?

Reply:
I mean, that in Mbed OS, we actually ignore this parameter, so to avoid confusion, we recommend this parameter being NULL. However, having it a different value will not have any effect, except that it will make the code less underastood

Reply:
Could we rephrase this as "We recommend you set the platform context parameter for these functions as `NULL`."?